mobileapp/src/components/screens/auth/LoginScreen/LoginScreen.tsx

```diff
@@ -64,6 +72,12 @@ export const LoginScreen: React.FC = () => {
             <Box h={10} />
             <Field name="password" component={FormikInput} type="password" />
             <Box h={10} />
+            <Field
+              name="hostUrl"
+              component={FormikInput}
+              autoCapitalize="none"
+            />
+            <Box h={10} />
             <Button isLoading={loading} onPress={() => handleSubmit()}>
               Login
```

Comment:
Trzeba wywalić stringi do plików. Czapiewski mówił że będzie za to oceny obniżał. Wydaje mi się że lepiej to zrobić od razu.

Reply:
mysle ze lepiej bedzie za jednym razem machnac wszystkie stringi, w takiej aplikacji to zajmie maks 2h
a docsy tez mysle ze na koniec, bo funkcje będą się zmieniać, i jeszcze trzeba bedzie uzgodnic typ docsów